PointOfDispensingApp/src/components/CreateLocationResourcePage.vue

```diff
@@ -0,0 +1,301 @@
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="5">
+        <v-row no-gutters>
+          <v-col cols="4">
+            <div class="secondary--text">City</div>
+          </v-col>
+          <v-col cols="8">
+            <v-text-field outlined dense 
+              v-model="search_city"
+            ></v-text-field>
+          </v-col>
+          <v-col cols="4">
+            <div class="secondary--text">Zip Code</div>
+          </v-col>
+          <v-col cols="8">
+            <v-text-field outlined dense 
+              v-model="search_postalCode"
+            ></v-text-field>
+          </v-col>
+        </v-row>
+      </v-col>
+      <v-col cols="1">
+        <!--blank column for spacing-->
+      </v-col>
+      <v-col cols="5">
+        <v-row no-gutters>
+          <v-col cols="4">
+            <div class="secondary--text">State</div>
+          </v-col>
+          <v-col cols="8">
+            <v-select
+              :items="stateOptions"
+              outlined
+              dense
+              v-model="search_state"
+            ></v-select>
+          </v-col>
+          <v-col cols="4">
+            <div class="secondary--text">Organization</div>
+          </v-col>
+          <v-col cols="8">
+            <v-text-field outlined dense 
+              v-model="search_organization"
+            ></v-text-field>
+          </v-col>
+        </v-row>
+      </v-col>
+    </v-row> 	
+    <v-row no-gutters>
+      <v-col cols="12">
+        <v-btn color="accent" @click="retrieveLocationResources()">
+          Search
+        </v-btn>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <v-divider></v-divider>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <template>
+          <v-data-table
+            @click:row="rowClick"
+            item-key="siteID"
+            single-select
+            :items="locationResourceLookupTable"
+            :headers="headers"
+            class="elevation-1"
+            :footer-props="{
+              'items-per-page-options':[5]
+            }"
+            :items-per-page="5"
+          ></v-data-table>
+        </template>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <v-dialog
+          v-model="createLocationResourceDialog"
+          persistent
+          max-width="70%"
+          >
+            <template v-slot:activator="{ on, attrs }">
+              <v-btn color="accent" v-bind="attrs" v-on="on">
+                Create location resource
+              </v-btn>
+            </template>
+            <v-card>
+              <v-card-title class="headline grey lighten-2 justify-center">
+                Create Location Resource
+              </v-card-title>
+              <v-card-text>
+                <v-container>
+                  <v-row no-gutters>
+                    <v-col cols="2">
+                      <div class="secondary--text">Site ID</div>
```

Comment:
Site Id is an internal tracker our software should set and not by the group. We should remove it here and move Site Name to its place.

Reply:
I can remove it, but we should probably confirm this assumption.  Counties may assign (or be assigned) some identifier that we may want to make use of.  If not, then I agree that we should auto-generate something.